package.json

```diff
@@ -77,8 +77,8 @@
           "order": 1
         },
         "eslintrcPath": {
-          "title": ".eslintrc Path",
-          "description": "It will only be used when there's no config file in project",
+          "title": ".eslintrc Absolute Path",
+          "description": "The absolute path to eslint config file that is used when there's no config file in project",
```

Comment:
This doesn't currently need to be an absolute path. A relative path works for me.

Reply:
Revertedd